source/org/zfin/framework/SessionUtil.java

```diff
@@ -0,0 +1,35 @@
+package org.zfin.framework;
+
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.NoSuchElementException;
+
+public class SessionUtil {
+
+    public static void setVariable(String name, Object value) {
```

Comment:
What do you need this class for? I understand you are storing info in session / request. What is this info?

Reply:
I use a session variable to store a message to display to the user after they are redirected from a delete fish page to a publication with errata.